src/qml/controls/IPAddressValueInput.qml

```diff
@@ -53,30 +53,4 @@ TextInput {
     Behavior on color {
         ColorAnimation { duration: 150 }
     }
-
-    function isValidIPPort(input)
-    {
-        var parts = input.split(":");
-        if (parts.length !== 2) return false;
-        if (parts[1].length === 0) return false; // port part is empty
-        var ipAddress = parts[0];
-        var ipAddressParts = ipAddress.split(".");
-        if (ipAddressParts.length !== 4) return false;
-        for (var i = 0; (i < ipAddressParts.length); i++) {
-            if (ipAddressParts[i].length === 0) return false; // ip group number part is empty
-            if (parseInt(ipAddressParts[i]) > 255) return false;
```

Comment:
Could have been negative and still considered as valid: `1.2.-3.4`

Reply:
Yeah, all that will be gone anyways.